routes/index.js

```diff
@@ -8,6 +8,10 @@ var assert = require('assert');
 var ObjectId = require('mongodb').ObjectID;
 var url = process.env.MONGODB_URI;
 
+router.get('/home', function(req, res) {
```

Comment:
I think the homepage should be "/" and the dashboard should be "/dashboard" or something similar